### DISCUSSION TITLE
not able to create openalgo account

### DISCUSSION THREAD

**Discussion by grvnanda** [marketcalls/openalgo#105] [General]
http://127.0.0.1:5000
I have setup everything, when I am clicking this link it doesnt open. I need to create openalgo account. currently It is showing error as Access to 127.0.0.1 was denied
You don't have authorisation to view this page.

**Comment by marketcalls** [comment]
Let me help by asking more relevant questions:

Could you share a screenshot of the exact error you're seeing when trying to connect to OpenAlgo?
Which operating system are you using? (Windows/Mac/Linux)
Which broker are you trying to connect to?
Are you running any antivirus or firewall software?

The error "Access to 127.0.0.1 was denied" suggests this is a local connection issue. This often happens when:

Your firewall is blocking the connection
The application doesn't have proper permissions to run
There might be a port conflict

Until I can see more details about your setup, here are some basic things to check:

Make sure you're running the application with proper permissions
Check if you have any other applications using the same port

**Reply by grvnanda** [reply]
Could you share a screenshot of the exact error you're seeing when trying to connect to OpenAlgo? 
Which operating system are you using? (Windows/Mac/Linux) I am using MAC
Which broker are you trying to connect to? FYERS
<img width="9" alt="Screenshot 2024-12-09 at 4 18 47 PM" src="https://github.com/user-attachments/assets/3a4a3a9c-8ed0-4429-bfb1-5a5141834693">

Are you running any antivirus or firewall software? NO

when I am clicking 127.0.0.1 there is no error and blank screen as attached.
<img width="1234" alt="Screenshot 2024-12-11 at 11 07 07 AM" src="https://github.com/user-attachments/assets/f90ba996-d79b-482d-a41b-9201ee3f0630">

**Comment by marketcalls** [comment]
you have to try at 127.0.0.1:5000 thats where the python flask app run by default

**Reply by grvnanda** [reply]
<img width="966" alt="Screenshot 2024-12-11 at 11 18 08 AM" src="https://github.com/user-attachments/assets/4495d9c9-ae59-42b2-955c-79dc81d04c5c">

it is 127.0.0.1:5000 when entered showing only 127.0.0.1. see screenshot

**Comment by marketcalls** [comment]
Did your  app started properly? can you verify in the console?
<img width="843" alt="Screenshot 2024-12-11 at 11 30 06 AM" src="https://github.com/user-attachments/assets/eeeced06-3284-4288-8765-861b7b2c6520">

**Reply by grvnanda** [reply]
no I tried it is not running.

**Reply by grvnanda** [reply]
![Uploading Screenshot 2024-12-11 at 1.22.37 PM.png…]()

**Reply by grvnanda** [reply]
<img width="729" alt="Screenshot 2024-12-11 at 2 43 55 PM" src="https://github.com/user-attachments/assets/84ae0584-d3c3-4512-af8e-9a174321dab7">

**Comment by marketcalls** [comment]
If you are running in Mac OS Please follow the procedure here
https://docs.openalgo.in/getting-started/mac-os-installation
